src/XrmDefinitelyTyped/Resources/Extensions/xrm_ext_9-.d.ts

```diff
@@ -499,9 +499,9 @@ declare namespace Xrm {
 
     interface WebResource {
         /**
-         * The type of the page.
+         * Specify "webresource".
          */
-        pageType: "webresource";
+        pageType: string;
```

Comment:
The typing "should" support specifying the string "webresource" here no?
This introduces a possibility of an unsupported value being passed, and would potentially make it harder for TSC to verify the type of the object if not explicitly specified

Reply:
Fixed